cypress/page/shipping-step.page.ts

```diff
@@ -5,7 +5,7 @@ class ShippingStepPage {
 
   constructor() {
       this.cartContent = "Proceed to checkout";
-      this.cartNav = ".cart_navigation";
+      this.cartNav = ".cart_navigation .button span";
```

Comment:
you don't need to change this selector, as you are using the get in combination with the contains

Reply:
Fixed